### DISCUSSION TITLE
Is Kloadgen compatible with Blazemeter

### DISCUSSION THREAD

**Discussion by smartist17** [sngular/kloadgen#385] [Q&A]
I am trying to integrate my JMX file into Blazemeter to conduct load testing. I am unable to establish a kafka connection. Is this Kloadgen plugin compatible with Blazemeter?

**Comment by jemacineiras** [comment]
Hi @smartist17,

thank you for asking. As far as we know blazemeter is base on Jmeter so in our guessing it should be 100% compatible. 
But if you have problems, please let us know opening and issue and we will try to help you.

Cheers
